src/Components/RosTable/RenderColumn.js

```diff
@@ -49,9 +51,22 @@ export const displayOS = (data) => {
 };
 
 export const displayLastReported = (data) => {
+    const daysAgoSeven = moment().subtract(7, 'days');
+    const isStale = moment(data).isBefore(daysAgoSeven);
+    const exactDate = dateStringByType('exact')(new Date(data));
+    const relativeDate = dateStringByType('relative')(new Date(data));
```

Comment:
```suggestion
    const lastReported = new Date(data);
    const exactDate = dateStringByType('exact')(lastReported);
    const relativeDate = dateStringByType('relative')(lastReported);
```
Instead of creating two separate date objects, will it be good to create single date object?

Reply:
I agree @kgaikwad 💯 
refactored the code and pushed the changes. 🚀